src/highscores/highscores.html

```diff
@@ -2,14 +2,87 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>BATTLESHIP High Scores</title>
+
 </head>
 
 <body>
-
+    <main class="main">
+        <h1 class="title">High scores</h1>
+        <table class="table">
+            <tr class="table__headings">
+                <th class="table__heading"></th>
+                <th class="table__heading">Name</th>
+                <th class="table__heading">Score</th>
+                <th class="table__heading">Status</th>
+            </tr>
+            <tr class="table__datas">
+                <td class="table__data">1</td>
+                <td class="table__data table__data-name"></td>
+                <td class="table__data table__data-score"></td>
+                <td class="table__data table__data-status"></td>
+            </tr>
+            <tr class="table__datas">
+                <td class="table__data">2</td>
+                <td class="table__data table__data-name"></td>
+                <td class="table__data table__data-score"></td>
+                <td class="table__data table__data-status"></td>
+            </tr>
+            <tr class="table__datas">
+                <td class="table__data">3</td>
+                <td class="table__data table__data-name"></td>
+                <td class="table__data table__data-score"></td>
+                <td class="table__data  table__data-status"></td>
+            </tr>
+            <tr class="table__datas">
+                <td class="table__data">4</td>
+                <td class="table__data table__data-name"></td>
+                <td class="table__data table__data-score"></td>
+                <td class="table__data table__data-status"></td>
+            </tr>
+            <tr>
+                <td class="table__data">5</td>
+                <td class="table__data table__data-name"></td>
+                <td class="table__data table__data-score"></td>
+                <td class="table__data table__data-status"></td>
+            </tr>
+            <tr>
+                <td class="table__data">6</td>
```

Comment:
One row is with class `table__datas` and another one with `table__data`. Something went wrong